version.gradle.kts

```diff
@@ -25,6 +25,6 @@
  * as we want to manage the versions in a single source.
  */
 
-val spineBaseVersion: String by extra("1.5.12")
+val spineBaseVersion: String by extra("1.5.18")
 val spineTimeVersion: String by extra("1.5.12")
```

Comment:
Let's migrate to the latest `time` once it's published.

Also, let's set the `core-java` version to be the same as `base`.